doc/user-guide.adoc

```diff
@@ -114,6 +114,11 @@ Each `RuntimeComponent` CR must at least specify the `applicationImage` paramete
 | `route.insecureEdgeTerminationPolicy`   | HTTP traffic policy with TLS enabled. Can be one of `Allow`, `Redirect` and `None`.
 | `route.certificate`  | A YAML object representing a link:++https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1alpha2.CertificateSpec++[Certificate].
 | `route.certificateSecretRef` | A name of a secret that already contains TLS key, certificate and CA to be used in the route. Also can contain destination CA certificate.
+| `affinity.nodeAffinity` | A YAML object representing a link:++https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#nodeaffinity-v1-core++[NodeAffinity].
```

Comment:
Please change to "A YAML object that represents" as per Karen's suggestion. There are more instances below.

Reply:
I tried to follow the patter we already have for other objects, I though it is good to be consistent